scripts/upgrade

```diff
@@ -45,6 +46,12 @@ fi
 # Add peer_port also on UDP.
 ynh_exec_warn_less yunohost firewall allow UDP $peer_port
 
+if [ -z "$datadir" ]; then
+	datadir=/home/yunohost.app/$app
+	ynh_app_setting_set --app=$app --key=datadir --value=$datadir
+	cp -a "/home/yunohost.transmission" "$datadir/$app"
```

Comment:
That would copy the files to `/home/yunohost.app/$app/$app`, right?

Reply:
You are right! And also I don't know if its the best way to transition from the old path to the new one...

Reply:
Yeah, it will be tricky in any case. Copying may eat away free space (maybe a `rsync -a --remove-source-files` would be better?), changing the directory may break users workflows...